core/domain/exp_domain.py

```diff
@@ -1277,6 +1277,13 @@ def get_all_html_content_strings(self):
                     rule_spec_html = rule_spec.inputs['x']
                     html_list = html_list + rule_spec_html
 
+        if self.id == 'ListOfSetsOfHtmlStrings':
```

Comment:
Is this supposed to be `DragAndDropSort?`?

Reply:
Not sure if this will solve your problem, but I noticed an inconsistency here against the ItemSelectionInput which uses SetOfHtmlString

Reply:
Fixed it but yes, it hasn't solved the problem.

Reply:
Done, just changed this id.